director/vms.go

```diff
@@ -109,26 +110,35 @@ func (d DeploymentImpl) VMInfos() ([]VMInfo, error) {
 		return nil, err
 	}
 
+	addTimestampToInfos(infos)
+
+	return infos, nil
+}
+
+func addTimestampToInfos(infos []VMInfo) {
 	t := time.Now()
 
 	for _, info := range infos {
 		info.Timestamp = t
 	}
-
-	return infos, nil
 }
 
 func (c Client) DeploymentVMInfos(deploymentName string) ([]VMInfo, error) {
+	return c.DeploymentResourceInfos(deploymentName, "vms")
+}
+
+func (c Client) DeploymentResourceInfos(deploymentName string, resourceType string) ([]VMInfo, error) {
```

Comment:
seems like DeploymentResourceInfos should be private given that only DeploymentVMInfos and DeploymentInstanceInfo should call it.